sdk/openai/inference/src/clients/openai_client.rs

```diff
@@ -0,0 +1,69 @@
+use std::sync::Arc;
```

Comment:
Microsoft policy is that all source files have copyright headers. See `azure_core` for an example.